### ISSUE TITLE
main price font size too big?

### ISSUE THREAD

**Issue by bertybassett** [pufferfish-tech/octopus-agile-pi-prices#11]
![IMG_1312](https://user-images.githubusercontent.com/5606175/92625258-56ac5a80-f2c0-11ea-9d86-348036439317.jpeg)

how can I reduce the size of the font?

**Comment by bertybassett** [comment]
I changed this line to 
font = ImageFont.truetype(FredokaOne, 55)

**Comment by pufferfish-tech** [comment]
Yeah it's a bit of a compromise, it looks good at that size when it's outside of "peak", but overlaps a bit. You could code round it I guess to use a smaller font if price > 9.9p. I'll add it to the list of things to do ;)

**Comment by bertybassett** [comment]
manually changing to a font size of 55 was fine by me.
